engine/src/main/java/org/terasology/engine/subsystem/lwjgl/LwjglGraphics.java

```diff
@@ -254,6 +254,7 @@ private void checkOpenGL() {
                 GLContext.getCapabilities().OpenGL12,
                 GLContext.getCapabilities().OpenGL14,
                 GLContext.getCapabilities().OpenGL15,
+                GLContext.getCapabilities().OpenGL20,
```

Comment:
Regarding shaders, I don't know enough about the current setup (getting there) but I would have thought it'd be easier to have two sets of semi-equivalent shaders than coming up with our own shader format. 

Regarding the opengl profiles, sounds good to me, but I'm wondering how realistic it is, given the human resources available, to support two renderers. When I wrote about replaceable renderers I just meant it to open the door to the possibility rather than Terasology actually having more than one to support. It'd be nice to switch to OpenGL core profile though. But yes, even just that would be a lot of work.